README.md

```diff
@@ -1,17 +1,31 @@
 # kuery
 
 Simple Mongo-like queries with lo-dash, at the moment the operators are very limited...
-```javascript
-var collection =  [
-	{id:1, name:'Andreas',address:{street:'Bellmansgatan'}},
-	{id:2, name:'Sven'},
-	{id:3, name:'Christian'},
-	{id:4, name:'Emil'},
+
+### Usage
+```ts
+import Kuery from 'Kuery';
+
+interface Person {
+	id: number;
+	name: string;
+	address?: Address;
+}
+
+interface Address {
+	street: string;
+}
```

Comment:
Just for the sake of convention, since we're using spaces rather than tabs.

```suggestion
interface Person {
  id: number;
  name: string;
  address?: Address;
}

interface Address {
  street: string;
}
```